Dockerfile

```diff
@@ -8,14 +8,21 @@ WORKDIR /go/src/github.com/openfaas/faas-swarm
 COPY vendor     vendor
 COPY handlers	  handlers
 COPY types      types
+COPY version    version
+COPY .git       .git
 COPY server.go  .
 
 RUN curl -sL https://github.com/alexellis/license-check/releases/download/0.2.2/license-check > /usr/bin/license-check \
     && chmod +x /usr/bin/license-check
 RUN license-check -path ./ --verbose=false "Alex Ellis" "OpenFaaS Project"
 
 RUN gofmt -l -d $(find . -type f -name '*.go' -not -path "./vendor/*") \
-  && CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o faas-swarm .
```

Comment:
You shouldn't have deleted this line as it may have changed the name of the output binary.

Reply:
I have edited it to inject the version and commit id.